sigma/validators/sigmahq/logsource.py

```diff
@@ -59,29 +59,24 @@ def validate(self, rule: SigmaRule) -> List[SigmaValidationIssue]:
             return []
 
 
-@dataclass
-class SigmahqLogsourceDefinitionIssue(SigmaValidationIssue):
-    description: ClassVar[str] = "Rule uses an unknown logsource definition"
-    severity: ClassVar[SigmaValidationIssueSeverity] = SigmaValidationIssueSeverity.MEDIUM
-    logsource: SigmaLogSource
-
-
-class SigmahqLogsourceDefinitionValidator(SigmaRuleValidator):
-    """Checks if a rule uses the unknown logsource definition."""
-
-    def validate(self, rule: SigmaRule) -> List[SigmaValidationIssue]:
-        if rule.logsource.definition:
-            core_logsource = SigmaLogSource(
-                rule.logsource.category, rule.logsource.product, rule.logsource.service
-            )
-            if (
-                core_logsource in config.sigma_taxonomy
-                and config.sigma_taxonomy[core_logsource]["logsource"]["definition"]
-            ):
-                if (
-                    rule.logsource.definition
-                    != config.sigma_taxonomy[core_logsource]["logsource"]["definition"]
-                ):
-                    return [SigmahqLogsourceDefinitionIssue(rule, rule.logsource)]
-
-        return []
+# @dataclass
```

Comment:
Better remove instead of commenting out, it's in the versioning history anyways.